Fix unittests failures introduced from D69212362

zhxchen17 opened the pull request:
Summary: There'll be some minor test failures caused by D69212362, fixing these before the source diff get landed.

Differential Revision: D69251115

larryliu0820 reviewed (approved):
Thanks for the pre-emptive fix

zhxchen17 committed:
Fix unittests failures introduced from D69212362 (#8278)

Summary:

There'll be some minor test failures caused by D69212362, fixing these before the source diff get landed.

Reviewed By: larryliu0820

Differential Revision: D69251115
